website/videoBlog/2024-02-18-Build-Incremental-ETL-pipeline-with-Hudi-and-Airflow-and-MinIO.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: "Build Incremental ETL pipeline with Hudi and Airflow and MinIO"
+last_modified_at: 2024-02-21T16:54:38.964863-07:00
+authors:
+- name: Soumil Shah
+category: blog
+image: /assets/images/video_blogs/2024-02-18-Build-Incremental-ETL-pipeline-with-Hudi-and-Airflow-and-MinIO.png
+navigate: "https://www.youtube.com/watch?v=PLYgUOzTnJ8"
+tags:
+- guide
+- beginner
+- apache hudi
+- minio
+- apache airflow
+- ETL
```

Comment:
Cleanup ETL related tags:
- Change to `etl`
- Rename `https://hudi.incubator.apache.org/videos/tags/etl-jobs` to `etl`
- Rename https://hudi.incubator.apache.org/videos/tags/batch-etl to `batch etl` instead of `batch ETL`
- Rename https://hudi.incubator.apache.org/videos/tags/incremental-etl/ from `incremental ETL` to `incremental etl`
- Rename https://hudi.incubator.apache.org/videos/tags/streaming-etl from `streaming-etl` to `streaming etl`

Reply:
Thanks for taking care of most of it. Still I find some occurrences of https://hudi.incubator.apache.org/videos/tags/incremental-etl/ which needs to be changed to `incremental-etl` tag. Please take care of it.